chromium_src/chrome/browser/about_flags.cc

```diff
@@ -9,6 +9,7 @@
 #include "brave/common/brave_features.h"
 #include "brave/common/pref_names.h"
 #include "brave/components/brave_component_updater/browser/features.h"
+#include "brave/components/brave_rewards/common/features.h"
```

Comment:
I think we'd need to add a dependency into //chrome/browser via `//brave/browser/sources.gni`. Seems like a number of deps are missing for other header includes here as well. Or is there another route for deps @bridiver ?

Reply:
Yes, when adding the feature flag I wasn't sure about how deps would play into it. Any advice appreciated!

Reply:
I think, I would put `brave/components/brave_rewards/common/features{.h|.cc}` into a separate `source_set("features")`. Then add `brave/components/brave_rewards/common/sources.gni` which would define `brave_components_brave_rewards_common_deps = [ "//brave/components/brave_rewards/common:features" ]`. Then include that .gni into `//brave/browser/sources.gni` and add `brave_components_brave_rewards_common_deps` to `brave_chrome_browser_deps`

Reply:
Is there another component that uses feature flags that does something similar? I'm pretty confused. Is there documentation for how to put this all together?

Reply:
I see that a similar thing is done with `brave_wallet` - I can use that for guidance as well. Thanks!

Reply:
Actually, I'm still confused 😄, since there isn't a `browser/brave_rewards/sources.gni` currently. It seems there's a bunch of things missing - not sure how to resolve in the context of this PR.

Reply:
I think maybe something like this: https://github.com/brave/brave-core/commit/be3038b32f196c8aaed7de89206b0e909011e212

Reply:
Ah, it makes sense now! 😄